azurerm/resource_arm_loadbalancer_rule.go

```diff
@@ -227,6 +233,10 @@ func resourceArmLoadBalancerRuleRead(d *schema.ResourceData, meta interface{}) e
 			d.Set("enable_floating_ip", properties.EnableFloatingIP)
 		}
 
+		if properties.DisableOutboundSnat != nil {
+			d.Set("disable_outbound_snat", properties.DisableOutboundSnat)
+		}
```

Comment:
_minor_ since this is a simple type (e.g. Bool/Int/String ..) we can remove this if-check here as `d.Set` will handle this being nil